content/200-concepts/100-components/02-prisma-client/230-handling-exceptions-and-errors.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Handling exceptions and errors'
+metaTitle: 'Handling exceptions and errors (Reference)'
+metaDescription: 'This page covers how to handle exceptions and errors'
+---
+
+<TopBlock>
+
+In order to handle diffferent types of errors you can use `instanceof` to check what the error is and handle it accordingly.
+
+The following example tries to create a user with an already existing email record. The email field has a unique identifier applied to it.
```

Comment:
```suggestion
The following example tries to create a user with an already existing email record. This will throw an error because the `email` field has a unique attribute applied to it.
```